packages/office-ui-fabric-react/src/components/ExtendedPicker/PeoplePicker/ExtendedPeoplePickerPage.tsx

```diff
@@ -8,7 +8,7 @@ import {
 import { ExtendedPeoplePickerTypesExample } from '../examples/ExtendedPeoplePicker.Basic.Example';
 
 const ExtendedPeoplePickerBasicExampleCode = require(
-  '!raw-loader!@uifabric/experiments/src/components/ExtendedPicker/examples/ExtendedPeoplePicker.Basic.Example.tsx'
```

Comment:
Is this still an extendedPeoplepicker? Or is should it be named something else to differentiate it from the existing people picker?

Reply:
It's still called the extended picker, not sure what a better name for this is. Maybe this can be addressed down the road if its really confusing for others.